diff --git a/node_modules/gatsby-remark-images/index.js b/node_modules/gatsby-remark-images/index.js
index 21d25eb..2cad55b 100644
--- a/node_modules/gatsby-remark-images/index.js
+++ b/node_modules/gatsby-remark-images/index.js
@@ -463,7 +463,7 @@ module.exports = function (_ref, pluginOptions) {
               rawHTML = ("\n    <span\n      class=\"" + imageWrapperClass + "\"\n      style=\"position: relative; display: block; margin-left: auto; margin-right: auto; max-width: " + presentationWidth + "px; " + (imageCaption ? "" : wrapperStyle) + "\"\n    >\n      " + rawHTML + "\n    </span>\n    ").trim(); // Wrap in figure and use title as caption
 
               if (imageCaption) {
-                rawHTML = ("\n  <figure class=\"gatsby-resp-image-figure\" style=\"" + wrapperStyle + "\">\n    " + rawHTML + "\n    <figcaption class=\"gatsby-resp-image-figcaption\">" + imageCaption + "</figcaption>\n  </figure>\n      ").trim();
+                rawHTML = ("\n  <figure class=\"gatsby-resp-image-figure\" style=\"" + wrapperStyle + "\">\n    " + rawHTML + "\n    <figcaption class=\"gatsby-resp-image-figcaption\" align=\"center\">" + imageCaption + "</figcaption>\n  </figure>\n      ").trim();
               }
 
               return _context3.abrupt("return", rawHTML);
